src/main/resources/com/google/api/codegen/java/http_stub.snip

```diff
@@ -55,7 +55,6 @@
           .setHttpMethod(HttpMethods.{@methodDescriptor.httpMethod.httpMethod})
           .setRequestFormatter(
               ApiMessageHttpRequestFormatter.<{@methodDescriptor.requestTypeName}>newBuilder()
-                  .setRequestInstance({@methodDescriptor.requestTypeName}.getDefaultInstance())
```

Comment:
This is no longer required and it has been removed in https://github.com/googleapis/gax-java/pull/537.